### PR TITLE
Update platform support matrix for Authorization and Resiliency

### PR DESCRIPTION
**Pull request by xuluna**
# Description
Update support matrix for Authorization and Resiliency after qualifications

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1012 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/_index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS | Supported Versions |
 |-|-|
-| Kubernetes    | 1.25, 1.26, 1.27 |
+| Kubernetes    | 1.25, 1.26, 1.27, 1.28 |
 | RHEL          |     7.x, 8.x     |
```
Comment (gallacher) [inline content/docs/authorization/_index.md:37]: We no longer specify OS support so RHEL and CentOS rows can be removed

File: content/docs/authorization/_index.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS | Supported Versions |
 |-|-|
-| Kubernetes    | 1.25, 1.26, 1.27 |
+| Kubernetes    | 1.25, 1.26, 1.27, 1.28 |
```
Comment (gallacher) [inline content/docs/authorization/_index.md:36]: Remove 1.25

File: content/docs/authorization/_index.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ The following diagram shows a high-level overview of CSM for Authorization with
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerMax         | PowerFlex | PowerScale |
 |---------------|:----------------:|:-------------------:|:----------------:|
-| Storage Array |PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx,<br> Unisphere 10.0, 10.0.1|    3.5.x, 3.6.x    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 |
+| Storage Array |PowerMax 2500/8500 PowerMaxOS 10 (6079) , PowerMaxOS 10.0.1 (6079) <br> PowerMax 2000/8000 - 5978.711.xxx, 5978.479.xxx,<br> Unisphere 10.0, 10.0.1|    3.5.x, 3.6.x, 4.0.x, 4.5    | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 |
```
Comment (gallacher) [inline content/docs/authorization/_index.md:46]: Please remove 3.5.x, 3.6.x for PowerFlex

File: content/docs/resiliency/_index.md [hunk 4]
```diff
@@ -41,7 +41,7 @@ CSM for Resiliency provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 | COP/OS            | Supported Versions |
 | ----------------- | :----------------: |
-| Kubernetes        | 1.25, 1.26, 1.27   |
+| Kubernetes        | 1.25, 1.26, 1.27, 1.28   |
```
Comment (gallacher) [inline content/docs/resiliency/_index.md:44]: Remove 1.25 along with RHEL/CentOS rows

File: content/docs/resiliency/_index.md [hunk 5]
```diff
@@ -52,7 +52,7 @@ CSM for Resiliency provides the following capabilities:
 {{<table "table table-striped table-bordered table-sm">}}
 |               | PowerFlex    | Unity XT                          | PowerScale                              | PowerStore                    |
 | ------------- | :----------: | :-------------------------------: | :-------------------------------------: | :---------------------------: |
-| Storage Array | 3.5.x, 3.6.x | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4 | 1.0.x, 2.0.x, 2.1.x, 3.0, 3.2 |
+| Storage Array | 3.5.x, 3.6.x, 4.0.x, 4.5 | 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 | OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4 | 1.0.x, 2.0.x, 2.1.x, 3.0, 3.2 |
```
Comment (gallacher) [inline content/docs/resiliency/_index.md:55]: Please remove 3.5.x, 3.6.x for PowerFlex
Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6
Replace Unity: 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 with 5.1.2, 5.2, 5.3

File: content/docs/authorization/_index.md (hunk 1 above)
Comment (xuluna) [reply content/docs/authorization/_index.md]: > We no longer specify OS support so RHEL and CentOS rows can be removed

Fixed

File: content/docs/authorization/_index.md (hunk 2 above)
Comment (xuluna) [reply content/docs/authorization/_index.md]: > Remove 1.25

Fixed

File: content/docs/authorization/_index.md (hunk 3 above)
Comment (xuluna) [reply content/docs/authorization/_index.md]: > Please remove 3.5.x, 3.6.x for PowerFlex Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6

Fixed

File: content/docs/resiliency/_index.md (hunk 4 above)
Comment (xuluna) [reply content/docs/resiliency/_index.md]: > Remove 1.25 along with RHEL/CentOS rows

Fixed

File: content/docs/resiliency/_index.md (hunk 5 above)
Comment (xuluna) [reply content/docs/resiliency/_index.md]: > Please remove 3.5.x, 3.6.x for PowerFlex Replace OneFS 8.1, 8.2, 9.0, 9.1, 9.2, 9.3, 9.4, 9.5 with OneFS 9.3, 9.4, 9.5.0.5, 9.5.0.6 Replace Unity: 5.0.5, 5.0.6, 5.0.7, 5.1.0, 5.1.2, 5.2, 5.3 with 5.1.2, 5.2, 5.3

Fixed

**Comment by gallacher** [comment]
Closing because changes in this PR have already been made

**Pull request closed without merging**